web/src/main/java/model/Account.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
+@Entity
+@Table
+public class Account {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
+    private User user;
+
+    private String nameOfAccount;
```

Comment:
These fields are inside of the class Account, so it is clear that it is the name **of account**. Rename this to name.